src/reason-parser/reason_pprint_ast.ml

```diff
@@ -6442,7 +6442,12 @@ let printer = object(self:'self)
                     (atom (".")))
                   (self#formatNonSequencyExpression e))
           else
-            Some (makeLetSequence (self#letList x))
+            Some
```

Comment:
this looks exactly like the `then` branch.

I'm surprised this doesn't break `let open Foo; bar;` for example?

Perhaps the `letop` case should be added to `isSeriesOfOpensFollowedByNonSequencyExpression`, then we'd get printing for free?

Reply:
Well, not exactly the same. The first branch results into extra braces: 
```
-      let* x = Js.Promise.resolve(42);
-      Js.Promise.resolve(x * 2);
+      {
+        let* x = Js.Promise.resolve(42);
+        Js.Promise.resolve(x * 2);
+      }
```

Reply:
I did make a small change to avoid code duplication

Reply:
I don't see how this is correct yet. I might be missing something.

Aren't we replacing ALL `open` calls with `Foo.( ...  )`?

Reply:
@anmonteiro 

Not exactly, but the solution wasn't perfect (it didn't work for nested local open modules). I've tried to improve on it with help of an extra attribute set by the parser.